CMakeLists.txt

```diff
@@ -176,6 +176,13 @@ if (COMPILER_CLANG)
     endif()
 endif ()
 
+# If compiler has support for -Wreserved-identifier. It is difficult to detect by clang version,
+# because there are two different branches of clang: clang and AppleClang.
+# (AppleClang is not supported by ClickHouse, but some developers have misfortune to use it).
```

Comment:
@alexey-milovidov Well it's not a bad thing after all :) http://evan.nemerson.com/2021/05/04/portability-is-reliability/